index.html

```diff
@@ -28,6 +26,138 @@ <h3 class="job-title">I'm a Software Developer</h3>
       </div>
       <p class="job">I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
     </section>
-  </main>
-</body>
-</html>
```

Comment:
Good job working on the mobile version 😃, but you need to fix some parts to complete this milestone 👍.

- [ ] Kindly make sure you stuck to the design as much as possible (e.g., `font`, colors, images, text, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 
> h1 : font-family: 'Crete Round';

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |![mob](https://user-images.githubusercontent.com/57408419/227952310-23bbc890-35e0-4978-94d6-1ecfc4564471.JPG)| |![mob01](https://user-images.githubusercontent.com/57408419/227952466-65e277df-c0a9-434a-b741-6ab12b2884ce.JPG)| |